### DISCUSSION TITLE
best recipe to combine png images into movie

### DISCUSSION THREAD

**Discussion by avilella** [tuonglab/dandelion#116] [General]
I've now sorted out how to create the 360 png images for the 3D visualization, but every recipe I've tried to combine them into a movie using ffmpeg has resulted in large mp4 file which does not display with vlc or other video players.

What's the recommended way of generating the movie? Maybe the fact that they are dpi=720 is affecting the results from older ffmpeg recipes in StackOverflow?

Thanks in advance

**Comment by zktuong** [comment]
So I added `-pix_fmt yuv420p` as described [here](https://superuser.com/questions/820134/why-cant-quicktime-play-a-movie-file-encoded-by-ffmpeg) and it worked for me. Unfortunately I didn't keep the full command I used; but I wouldn't have deviated much and would have used something close to what's on the linked page.

**Comment by avilella** [comment]
Thanks, the one below worked:
```
ffmpeg -pattern_type glob -i '*.png' -vcodec libx264 -vf scale=640:-2,format=yuv420p out.mp4
```
